modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/podcast/PodcastFragment.kt

```diff
@@ -656,7 +658,12 @@ class PodcastFragment : BaseFragment(), Toolbar.OnMenuItemClickListener, Corouti
     }
 
     private fun FragmentPodcastBinding.setupMultiSelect() {
+        viewModel.multiSelectEpisodesHelper = multiSelectEpisodesHelper
+        viewModel.multiSelectBookmarksHelper = multiSelectBookmarksHelper
+        sharedBookmarksViewModel.multiSelectHelper = multiSelectBookmarksHelper
```

Comment:
When resolving merge conflicts, I had to determine where to put these assignments. I'm pretty confident this is the right place, but it's worth taking a close look here to double-check.

Reply:
This looks good to me 👍